drivers/interrupt_controller/intc_intel_vtd.h

```diff
@@ -47,7 +47,7 @@ struct vtd_irte {
 #define IRTA_SIZE 7  /* size = 2^(X+1) where IRTA_SIZE is X 2^8 = 256 */
 
 struct vtd_ictl_data {
-	struct vtd_irte irte[IRTE_NUM];
+	struct vtd_irte irte[IRTE_NUM]__attribute__ ((aligned (4096)));
```

Comment:
I believe you should be using the `__aligned` macro instead (which will then translate to the correct toolchain/compiler version)

Reply:
There is a bug in this original struct that you did not catch, which makes the whole thing crashing btw...